pkg/transport/transport.go

```diff
@@ -68,7 +68,7 @@ func (transport *Transport) DialNode(ctx context.Context, node *pb.Node, opts ..
 		return nil, Error.Wrap(err)
 	}
 
-	options := append([]grpc.DialOption{dialOpt}, opts...)
+	options := append([]grpc.DialOption{dialOpt, grpc.WithBlock()}, opts...)
```

Comment:
Dial is called below as well. Also there were places where grpc.WithBlock is used explicitly, so those can be removed.

Reply:
Removed other WithBlock() instances in the codebase, and updated the DialAddress function to use the WithBlock() opts as well.